Microsoft.Toolkit.Uwp.UI.Controls.Core/Microsoft.Toolkit.Uwp.UI.Controls.Core.csproj

```diff
@@ -63,4 +61,5 @@
     <Message Text="CSFiles: @(GeneratedCSFiles->'&quot;%(Identity)&quot;')" />
```

Comment:
So this was removed in the Sample App, but not here? So trying to understand where we think we need it or not, or was the sample app change a mistake?

Did you use some tooling to run across the project files or was the sample app one just a find/replace?

Reply:
I removed it in the App project to test whether it builds successfully. I have moved this logic to a separate targets file in #3894.